src/Wallabag/CoreBundle/Form/Type/EntryType.php

```diff
@@ -10,7 +10,7 @@ class EntryType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('url', 'url')
+            ->add('url', 'url', array('required' => true))
```

Comment:
And same here, the url field in the entity can not be blank `@Assert\NotBlank()`